### ISSUE TITLE
What improvements have been made for v3.2 Super model?

### ISSUE THREAD

**Issue by Sanster** [imoneoi/openchat#49]
Thank you for releasing the new model, I would like to know what improvements have been made to this Super model? Thanks

**Comment by imoneoi** [comment]
Mainly hyperparameter tuning and a slight modification in the loss function. We'll update the README later.

**Comment by Sanster** [comment]
Thank you, I'm looking forward to your sharing. I have another question, could you share how many steps the v3.2 model has trained under the configuration of batch_size=64, epochs=5? (Because there is padding-free fine-tuning, the number of steps should be much less)

**Comment by imoneoi** [comment]
Okay, about 3K steps. The number of steps should be almost equal to `num_epochs * total_tokens / tokens_per_batch` since the packaging efficiency is always greater than 99%.
